data/tutorials/lg_00_data_types_and_matching.md

```diff
@@ -83,6 +83,9 @@ type person = { first_name : string; surname : string; mutable age : int; }
 val birthday : person -> unit = <fun>
 ```
 
+Please note: in the above example, we used the UK spelling of *colour*, but it will also
+work with the US spelling *color*.
+
```

Comment:
I think emphasizing that either US or UK spelling works is probably less helpful than emphasizing that "colour" is just the name of the type and we could choose any name for the type (though, it helps to choose a name that helps you remember what kind of data the type intends to formalize). Point being that we don't want people to wonder about UK or US english, but instead understand that it's just a name.

```suggestion
Please note: in the above example, we chose "colour" as the name of a type that holds RGB-values. If you wanted to, you could also use the US spelling "color".
```